backend/globalConfig.json

```diff
@@ -1 +1 @@
-{"mongoUri":"mongodb://127.0.0.1:58014/jest?","mongoDBName":"jest"}
+{"mongoUri":"mongodb://127.0.0.1:59903/jest?","mongoDBName":"jest"}
```

Comment:
Is this file auto-generated? Is it worth gitignoring it?

Reply:
agree